src/app/pages/assessment/questionnaire-selection/questionnaire-selection.component.ts

```diff
@@ -33,7 +37,24 @@ export class QuestionnaireSelectionComponent {
 
     this.questionnaireService
       .getQuestionnaires({ filter })
-      .subscribe((questionnaires) => (this.foundQuestionnaires = questionnaires.edges.map((e) => e.node)));
+      // Added filter to filter out the archieved quest.
+      .subscribe((questionnaires) => (this.foundQuestionnaires = questionnaires.edges.map((e) => e.node).filter(e => e.status !== 'ARCHIVED')));
```

Comment:
I think this filter should be done in back end because it might happen that all the list of 10 questionnaires that are received from back end are archived and there won't be shown anything